src/examples/assetMetadata.ts

```diff
@@ -0,0 +1,169 @@
+import { MetadataEntry } from '@polymeshassociation/polymesh-sdk/internal';
```

Comment:
This wasn't exported out of `api/entities/types` for which I raised a separate PR in SDK - https://github.com/PolymeshAssociation/polymesh-sdk/pull/864. Once the PR will be merged we can bump the SDK version here and change the import